scripts/mail-alarm

```diff
@@ -480,6 +491,46 @@ class HAHostFailedETG(EmailTextGenerator):
 
         return body_pattern % self.text
 
+class SRIOThroughputTotalAlertETG(EmailTextGenerator):
+  def __init__(self, cls, obj_uuid, value, alarm_trigger_period, alarm_trigger_level, sr_uuid, mail_language):
+        super(SRIOThroughputTotalAlertETG, self).__init__(mail_language, 'sr_io_throughput_total_alert')
+        if alarm_trigger_period is None:
+            alarm_trigger_period = 60
+        if cls == 'Host':
+            self.params = get_host_params(obj_uuid)
+        else:
+            raise Exception, "programmer error"
+        self.value = value
+        self.alarm_trigger_period = alarm_trigger_period
+        self.alarm_trigger_level = alarm_trigger_level
+        self.sr_uuid = sr_uuid
```

Comment:
`sr_uuid` is not used anywhere else, there is no need to store it in the class. I wonder if it would be the case to add it to the message body (e.g. `storage repository "%s" (uuid: "%s")` ...)